reana_cluster/configurations/reana-cluster.yaml

```diff
@@ -3,7 +3,7 @@ cluster:
   version: "v1.9.4"
   db_config: &db_base_config
     - REANA_SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://reana:reana@db:5432/reana"
-  root_path: "/tmp/reana"
+  root_path: "/reana"
```

Comment:
I  am setting a default here so we don't hardcode the path we are using in prod. We will have to always have a `reana-cluster.yaml` with `root_path` set and `db_persistence_path` when we don't use DBoD.